### PR TITLE
Add imprint and privacy policy page

### PR DESCRIPTION
**Pull request by mc388**

### CODE REVIEW

File: resource/view/layout/_partials/footer.twig [hunk 1]
```diff
@@ -14,7 +14,8 @@
                     <div class="widget widget_links clearfix">
                         <h4>{% trans %}About Eventerza{% endtrans %}</h4>
                         <ul>
-                            <li><a href="#">AGB</a></li>
+                            <li><a href="{{ path_for('staticPages.imprint', {'lang': page.lang}) }}">{% trans %}Imprint{% endtrans %}</a></li>
```
Comment (Sascha89) [inline resource/view/layout/_partials/footer.twig]: Did you enter the german translation?

File: resource/view/layout/_partials/footer.twig [hunk 2]
```diff
@@ -14,7 +14,8 @@
...
+                            <li><a href="{{ path_for('staticPages.privacyPolicy', {'lang': page.lang}) }}">{% trans %}Privacy Policy{% endtrans %}</a></li>
```
Comment (Sascha89) [inline resource/view/layout/_partials/footer.twig]: Did you enter the german translation?

File: resource/view/layout/_partials/footer.twig (hunk 1 above)
Comment (mc388) [inline resource/view/layout/_partials/footer.twig]: yes

File: resource/view/layout/_partials/footer.twig (hunk 2 above)
Comment (mc388) [inline resource/view/layout/_partials/footer.twig]: yes

**Pull request merged**